src/files/header/Header.js

```diff
@@ -48,16 +54,20 @@ class Header extends React.Component {
         </div>
 
         <div className='mb3 flex justify-between items-center bg-snow-muted joyride-files-add'>
-          <BarOption title={t('blocks')}>
-            { repoNumObjects ? SimplifyNumber(repoNumObjects, { decimal: 0 }) : 'N/A' }
+          <BarOption title={t('files')} isLink onClick={() => { onNavigate('/files') }}>
+            { filesSize ? humanSize(filesSize || 0) : 'N/A' }
```

Comment:
Let's move checks responsible for `'N/A'` and `|| 0`  to `humanSize`, so its not repeated  here and in line 70